WinUIGallery/App.xaml.cs

```diff
@@ -66,42 +64,30 @@ where module.FileName.EndsWith("Microsoft.WindowsAppRuntime.Insights.Resource.dl
             }
         }
 
-        // Get the initial window created for this app
-        // On UWP, this is simply Window.Current
-        // On Desktop, multiple Windows may be created, and the StartupWindow may have already
-        // been closed.
+        /// <summary>
+        /// Get the initial window created for this app.
+        /// </summary>
         public static Window StartupWindow
         {
-            get
-            {
-                return startupWindow;
-            }
+            get => startupWindow;
         }
+
         /// <summary>
-        /// Initializes the singleton Application object.  This is the first line of authored code
+        /// Initializes the singleton Application object. This is the first line of authored code
         /// executed, and as such is the logical equivalent of main() or WinMain().
         /// </summary>
         public App()
         {
-            this.InitializeComponent();
-
-#if WINUI_PRERELEASE
-            this.Suspending += OnSuspending;
-            this.Resuming += App_Resuming;
-            this.RequiresPointerMode = ApplicationRequiresPointerMode.WhenRequested;
-#endif
-        }
-
-        public void EnableSound(bool withSpatial = false)
-        {
-            ElementSoundPlayer.State = ElementSoundPlayerState.On;
```

Comment:
This still works today in WinUI, right? There's a setting in Settings to turn this on/off

Reply:
The function is not being used and the setting works without it.